locale/fi_FI/emailTemplates.xml

```diff
@@ -1,837 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Are you sure you want to strip this file of all its content?

Reply:
wonder if the translation plugin is exporting those correctly?